### PR TITLE
Add effects for Mentor of Legends and Siphon Torment, and tag several ancestral spell feats

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: packs/feat-effects/effect-mentor-of-legends.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "name": "Effect: Mentor of Legends",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Mentor of Legends]</p>\n<p>The target gains temporary Hit Points equal to half your level.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
```
Comment (simonward) [inline packs/feat-effects/effect-mentor-of-legends.json:13]: Should this be 10 minutes?

File: packs/feat-effects/effect-siphon-torment.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "name": "Effect: Siphon Torment",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Siphon Torment]</p>\n<p>The target gains temporary Hit Points equal to your level.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
```
Comment (simonward) [inline packs/feat-effects/effect-siphon-torment.json:13]: Should this be 5 minutes?

File: packs/feat-effects/effect-azarketi-purification.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
                 "key": "FlatModifier",
                 "label": "Azarketi Purification (vs. ongoing saves against poisons already in your body)",
                 "predicate": [
-                    "poison"
+                    "item:trait:poison"
```
Comment (simonward) [inline packs/feat-effects/effect-azarketi-purification.json:28]: Probably need another term in the predicate like `ongoing-poison`, as it doesn't grant the bonus against new poisons.

Comment (simonward) [reply packs/feat-effects/effect-azarketi-purification.json:28]: And the custom label may not be needed?

File: packs/feats/mask-of-power.json [hunk 4]
```diff
@@ -28,7 +28,51 @@
...
+                        "label": "PF2E.SpecificRule.Spell.PhantomPain.Label",
+                        "value": "phantom-pain"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Spell.SureStrike.Label",
+                        "value": "sure-strike"
+                    }
+                ],
+                "toggleable": true
+            },
```
Comment (TikaelSol) [inline packs/feats/mask-of-power.json:52]: Hold off on doing these daily prep ones. I don't want to add a bunch of spell names to localization files unless necessary

**Pull request merged**